proto/pulumi/provider.proto

```diff
@@ -384,6 +387,24 @@ message ConfigureRequest {
     // [](pulumirpc.ResourceProvider.Delete) calls. If true, the provider should expect these fields to be populated in
     // these calls. *Must* be true if the caller has previously called [](pulumirpc.ResourceProvider.Handshake).
     bool sends_old_inputs_to_delete = 6;
+
+    // The ID of the provider being configured. N.B. This will be null for engines before
+    // v3.155.0.
```

Comment:
I wonder if it's worth calling out also that this will be `null` if and only if `configure_with_urn` was `false` in a `Handshake`.

Reply:
Sure